Session03/Session03/Session03Excercise02/Program.cs

```diff
@@ -0,0 +1,31 @@
+﻿using System;
+using System.Linq.Expressions;
+
+namespace Exercise03
+{
+    class Program
+    {
+        static void Main(string[] args)
+        {
+            Console.WriteLine("Ange ett antal siffror, separerat med kommatecken.");
+            string input = Console.ReadLine();
+            string[] inputArray = input.Split(",", StringSplitOptions.RemoveEmptyEntries);
+            double?[] numberArray = new double?[inputArray.Length];
+
+            for (int i = 0; i < inputArray.Length; i++)
+            {
+                try
+                {
+                    numberArray[i] = Convert.ToDouble(inputArray[i]);
+                }
+                catch (Exception)
+                {
+                    numberArray[i] = null;
+                }
+            }
+
+            //Console.WriteLine("Värdet är " + number);
```

Comment:
Prova att göra klart denna övning.
Man behöver loopa igenom `numberArray` igen med en ytterligare for-loop.
Sedan behöver du hålla koll på `minValue`, `maxValue`, `sum` och `count`.
`count` skall inte ökas och `sum` skall inte adderas `if (numberArray[i] == null)`

Reply:
För att hoppa vidare i en for-loop kan man använda nyckelordet `continue`.
För att komma åt värdet av en nullbar `double` så behöver man skriva `numberArray[i].Value`